target/linux/omap/config-5.4

```diff
@@ -400,14 +409,6 @@ CONFIG_MMC_OMAP_HS=y
 CONFIG_MODULES_USE_ELF_REL=y
 CONFIG_MSDOS_FS=y
 CONFIG_MTD_CMDLINE_PARTS=y
-CONFIG_MTD_M25P80=y
-CONFIG_MTD_NAND=y
```

Comment:
This disables NAND support. Are you sure?
[Commit](https://github.com/torvalds/linux/commit/72c5af00272339af6bbed6fe7275cd731f57be2d#diff-f42d611a391dbc954692f5a7276d7785) renamed this symbol to CONFIG_MTD_RAW_NAND

Reply:
That wasn't the intention, `make kernel_oldconfig` just didn't prompt for these knobs. Fixed, thanks!